cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -155,6 +155,14 @@ public BuildCommand() {
     @CommandLine.Option(names = "--cloud", description = "Enable cloud artifact generation")
     private String cloud;
 
+    @CommandLine.Option(names = "--runtime-management-included", description = "package service management tools in " +
+            "the executable JAR file(s).")
+    private Boolean runtimeManagementIncluded;
+
+    @CommandLine.Option(names = "--enable-service-publish", description = "package service management tools " +
+            "for WSO2 API manager in the executable JAR file(s).")
```

Comment:
This needs to be updated. This has to give the message "Include ballerina service management tools for publishing service data to external service catalogs."

Reply:
Can we have one flag for both? Runtime config will decide different options.
ex. enableRemoteManagement

Reply:
No, we can't, with this option, we need to enable OAS generation for http services. For other options, this step is not required.